manifests/install.pp

```diff
@@ -1,12 +1,15 @@
-class psad::install inherits psad::params {
-  package { $psad::install::package:
-    ensure => present,
-    notify => Exec['psad_signature_update']
+class psad::install {
+  include psad
+  if $psad::manage_package {
+    package { $psad::package:
+      ensure => present,
+    }
   }
 
   exec { 'psad_signature_update':
-    command     => $signature_update_command,
+    command     => $psad::signature_update_command,
     returns     => [0,1],
-    refreshonly => true
+    refreshonly => true,
+    subscribe   => Package[$psad::package],
```

Comment:
Won't this mean that the signature update won't happen when `manage_package` is false?

Reply:
As long as the package is being managed *somewhere*, the signature update will happen.

For instance, I had to compile my own packages for Amazon Linux and Ubuntu.  I'm installing via S3 download because we don't yet have our own managed repository.  The EPEL copy of psad is uninstallable due to version incompatibilities, and in the process of compiling an Ubuntu version, I discovered an [unfixed bug](https://github.com/mrash/psad/issues/56) in the codebase.

But because I'm still installing it via the [Package](https://puppet.com/docs/puppet/4.10/types/package.html) resource, it will still trigger a signature update.  If I were installing via [Exec](https://puppet.com/docs/puppet/4.10/types/exec.html), I'd have to add a [notify](https://puppet.com/docs/puppet/4.10/metaparameter.html#notify) attribute.

Reply:
That makes sense- can you update the README with this new property?

Reply:
Sure; gimme a 'sec.

*(...clickety...)*

Done.